examples/sampleCalls.cpp

```diff
@@ -1044,10 +1051,10 @@ TEST_SUITE("Single Proof Verifier") {
         auto vk_path = params_dir + std::string("/darlin_csw_test_vk");
 
         // Test all cases
-        create_verify_csw_proof(true, true, proof_path, pk_path, vk_path);
-        create_verify_csw_proof(true, false, proof_path, pk_path, vk_path);
-        create_verify_csw_proof(false, true, proof_path, pk_path, vk_path);
-        create_verify_csw_proof(false, false, proof_path, pk_path, vk_path);
+        create_verify_csw_proof(true, true, proof_path, pk_path, vk_path, true);
+        create_verify_csw_proof(true, false, proof_path, pk_path, vk_path, true);
+        create_verify_csw_proof(false, true, proof_path, pk_path, vk_path, true);
+        create_verify_csw_proof(false, false, proof_path, pk_path, vk_path, true);
```

Comment:
Can we add another test case "Proof Verifier: CSWNoConst -Darlin" and pass the last flag to false (like I did for Cert) ?